heartbeat/nfsserver

```diff
@@ -454,10 +454,6 @@ nfsserver_monitor ()
 
 prepare_directory ()
 {
-	if [ -z "$fp" ]; then
```

Comment:
We shouldnt remove this, or it will try to populate `/`.

You can add `if ! ocf_is_probe && [ -z "$fp" ]` to `nfsserver_validate()` to avoid this.

Reply:
aha, you are right. My brain is messed in the late evening to copy&paste the code. I intended to set the empty $fp to /var/lib/nfs instead.

My impression of nfsserver_validate() is it is called for every monitor operation. Might be no need to check so frequently.